.github/workflows/ci.yml

```diff
@@ -28,13 +35,7 @@ jobs:
     - name: Extract libraries
       run: sudo unzip -j libs.zip -d /lib
 
-    - name: Install latest Rust stable
-      uses: actions-rs/toolchain@v1
-      with:
-        profile: minimal
-        toolchain: stable
-        override: true
-
+    - run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
```

Comment:
I'd rather use `actions-rs/toolchain` for this, just replace the `stable` in `toolchain` with `${{ matrix.toolchain }}`.